Sources/DatadogObjc/DatadogConfiguration+objc.swift

```diff
@@ -471,6 +484,19 @@ public class DDConfigurationBuilder: NSObject {
         _ = sdkBuilder.set(encryption: DDDataEncryptionBridge(objcEncryption: encryption))
     }
 
+    @objc
+    public func set(tracingHeaderType: DDTracingHeaderType) {
+        _ = sdkBuilder.set(tracingHeaderType: tracingHeaderType.swiftType)
+    }
+
+//    @objc
+//    public func set(tracingHeaderTypes: Set<DDTracingHeaderType>) {
+//        _ = sdkBuilder.set(
+//            tracingHeaderTypes: Set(tracingHeaderTypes
+//                .compactMap { $0.swiftType })
+//        )
+//    }
```

Comment:
This should be exposed right?

Reply:
Yes sorry. I was testing stuff between my branch and the shopist app and had to temporarily comment out some code.

I'll fix it in a minute :)